rclcpp/complex_msg_subscriber/main.cpp

```diff
@@ -26,8 +26,9 @@ class complex_msg_subscriber_cpp_node : public rclcpp::Node
   complex_msg_subscriber_cpp_node()
   : Node("complex_msg_subscriber_cpp")
   {
+    rclcpp::QoS qos(10);
     subscription_ = this->create_subscription<complex_msgs::msg::NestedMsgTest>(
-      "complex_msgs_msg_NestedMsgTest",
+      "complex_msgs_msg_NestedMsgTest", qos,
```

Comment:
Possible to use rclcpp::SystemDefaultsQoS() instead?

Reply:
Changed!